rust/vint64/src/lib.rs

```diff
@@ -73,7 +73,7 @@
 
 #![no_std]
 #![doc(html_root_url = "https://docs.rs/vint64/1.0.0")]
-#![forbid(unsafe_code)]
+// #![forbid(unsafe_code)]
```

Comment:
Let's go with the `unsafe` solution as this function is definitely a hotspot in Veriform, which uses it to calculate the length of all of the length prefix tags in its TLV encoding.

Can you delete this line?